pyproject.toml

```diff
@@ -0,0 +1,47 @@
+[project]
+name = "cookiecutter-python-template"
+authors = [
+    {name = "Reinout van Rees", email = "reinout@vanrees.org"},
+]
+description = "Cookiecutter template for a plain python project or library"
+readme = "README.rst"
+license = {text = "MIT"}
+version = "0.5.dev0"
+classifiers = [
+        "Development Status :: 4 - Beta",
+        "Environment :: Console",
+        "Intended Audience :: Developers",
+        "Natural Language :: English",
+        "License :: OSI Approved :: BSD License",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 2",
```

Comment:
Ouch, some cleanup afterwards can't hurt. python2 has gone the way of the dodo. I'll put  it on my todo list.